etcd3/locks.py

```diff
@@ -46,7 +52,7 @@ def __init__(self, name, ttl=60,
         # need to compare
         self.uuid = uuid.uuid1().bytes
 
-    def acquire(self, timeout=10):
+    def acquire(self, timeout=DEFAULT_TIMEOUT, raise_on_timeout=False):
```

Comment:
Could you make timeout default to `None` instead? That's more in line with the rest of the defaults.

Reply:
I considered that, but it would be a change in behavior - current default is 10s, and None means 'forever'.  That said, I'm okay with it, and will change it if you're sure.